Casks/pktriot.rb

```diff
@@ -7,5 +7,11 @@
   desc "Instantly and securely host server applications and static websites"
   homepage "https://packetriot.com/"
 
+  livecheck do
+    url "https://packetriot.com/downloads"
+    strategy :page_match
+    regex(%r{href=.*?/pktriot-(\d+(?:\.\d+)*)\.macos.tar.gz}i)
```

Comment:
@JoshuaMulliken 
We use `[._-]` instead of `-` in regex, also use `.t` for all tarballs, and also escape the last `.`
```suggestion
    regex(%r{href=.*?pktriot[._-](\d+(?:\.\d+)*)\.macos\.t}i)
```